### PR TITLE
Break before and after migrations into separate files

### PR DESCRIPTION
**Pull request by apmasell**
This intentionally drops `addIdentity` which is no longer required

### CODE REVIEW

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_delete_pool.sql [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  
+  SELECT ROW_COUNT() AS number_deleted;
+  
+  COMMIT;
+END//
+
+DELIMITER ;
+-- EndNoTest
+
+DROP VIEW IF EXISTS CompletedPartitions;
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_delete_pool.sql]: Are these views intended to be in the `delete_pool` file?

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_trigger_box.sql [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    '',
+    NEW.lastModifier,
+    'Box created.'
+  )//
+
+DELIMITER ;
+-- EndNoTest
+
+DROP VIEW IF EXISTS CompletedPartitions;
+CREATE OR REPLACE VIEW RunPartitionsByHealth AS
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_trigger_box.sql]: These non-box things (here down to SampleBoxPosition) look to be mis-categoried...

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_trigger_box.sql [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  AND bp.targetId = d.dilutionId;
+
+CREATE OR REPLACE VIEW PoolBoxPosition
+AS SELECT p.poolId, bp.boxId, bp.position
+FROM Pool p
+JOIN BoxPosition bp
+  ON bp.targetType = 'POOL'
+  AND bp.targetId = p.poolId;
+
+CREATE OR REPLACE VIEW PoolableElementView
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_afterMigrate/05_trigger_box.sql]: Also the PoolableElementView

File: sqlstore/src/main/resources/db/migration_afterMigrate/05_delete_pool.sql (hunk 1 above)
Comment (apmasell) [reply sqlstore/src/main/resources/db/migration_afterMigrate/05_delete_pool.sql]: Nope.

**Comment by apmasell** [comment]
Fixed.

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_library.sql [hunk 4]
```diff
@@ -0,0 +1,64 @@
+-- StartNoTest
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_library.sql]: Many of the add X files actually add a type X, but it seems like these and the ones in the `add_box` file actually add Ys and Zs. Maybe rename them to `add_library_adjacent` or `add_related_to_library`?

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_seqparam.sql [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  END IF;
+  
+  IF NOT EXISTS (SELECT 1 FROM SequencingParameters WHERE name = iName AND platformId = platId) THEN
+    SET createUser = getAdminUserId();
+    INSERT INTO SequencingParameters(name, platformId, xpath, readLength, paired, createdBy, creationDate, updatedBy, lastUpdated)
+    VALUES (iName, platId, iXpath, iReadLength, iPaired, createUser, createTime, createUser, createTime);
+  END IF;
+END//
+
+END//
```
Comment (callunity) [inline sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_seqparam.sql]: Remove this extra `END//`

**Comment by callunity** [comment]
I confirmed that there are 37 triggers before applying these changes, and 37 triggers after applying them :)

File: sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_library.sql (hunk 4 above)
Comment (apmasell) [reply sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_library.sql]: Okay. `add_library_parameters`. I didn't necessarily intend a 1:1 correspondence between procedures and files.

**Comment by callunity** [comment]
Hmm, maybe to `05_view_order_completions.sql` but this does concern order completions, which are separate from orders.

**Pull request merged**